### ISSUE TITLE
Cannot read property 'enabled' of undefined

### ISSUE THREAD

**Issue by Vad1mo** [schwamster/serverless-certificate-creator#3]
I have a custom certificate but manage the domain with cloudformation and not with the domain plugin 

This is the error I get:

```
TypeError: Cannot read property 'enabled' of undefined
    at CreateCertificatePlugin.evaluateEnabled (/Users/v/Development/si/kickstart/static-website-aws/node_modules/serverless-certificate-creator/index.js:58:65)
    at CreateCertificatePlugin.initializeVariables (/Users/v/Development/si/kickstart/static-website-aws/node_modules/serverless-certificate-creator/index.js:29:27)
    at CreateCertificatePlugin.createCertificate (/Users/v/Development/si/kickstart/static-website-aws/node_modules/serverless-certificate-creator/index.js:99:10)
    at BbPromise.reduce (/usr/local/lib/node_modules/serverless/lib/classes/PluginManager.js:390:55)```

**Comment by schwamster** [comment]
@Vad1mo I am not sure if I understand the problem. Did you create your domain and the certificate yourself?

**Comment by schwamster** [comment]
@Vad1mo fyi: the domain plugin is not mutually exclusive with cloudformation, terraform or any other mean of creating your a-records.  the domain plugin can simply be used to create the necessary base path mapping with an already existing domain.

**Comment by schwamster** [comment]
can you tell me how your custom/customCertificate serverless config looks like (please without sensitive data )

**Comment by Vad1mo** [comment]
Here is the config I am using.


```
....
plugins:
  - serverless-plugin-cloudfront-lambda-edge
  - serverless-finch
  - serverless-certificate-creator

custom:
  defaultRegion: us-east-1
  defaultEnvironmentGroup: dev
  region: ${opt:region, self:custom.defaultRegion}
  stage: ${opt:stage, 'poc'}
  objectPrefix: '${self:service}-${self:custom.stage}'
  client:
    bucketName: '${self:custom.objectPrefix}'
    distributionFolder: website/
  customCertificate:
    certificateName: 'docs.sandbox.com'
    hostedZoneName: 'sandbox.com.'
    region: ${self:custom.region}
  customDomain:
    enabled: true
....
```

**Comment by schwamster** [comment]
ah snap... now i understand what the problem is :)
copy paste bug 

will fix it asap!  thanks for pointing it out

**Comment by schwamster** [comment]
just published 1.0.4 => https://www.npmjs.com/package/serverless-certificate-creator/v/1.0.4
@Vad1mo  can you verify that it works for you?

**Comment by Vad1mo** [comment]
Works great thank you. ❤️
